src/Refinery/String/CaseOfLabelIfPossible.php

```diff
@@ -0,0 +1,279 @@
+use Throwable;
+
+/**
+ * Class CaseOfLabelIfPossible
+ *
+ * Format a text for the title capitalization presentation (Specification at https://docu.ilias.de/goto_docu_pg_1430_42.html)
+ *
+ * Throws a LanguageNotSupportedException in the transform method, if a not supported language is passed
+ *
+ * @package ILIAS\Refinery\String
+ *
+ * @author  studer + raimann ag - Team Custom 1 <support-custom1@studer-raimann.ch>
+ */
+class CaseOfLabelIfPossible implements Transformation {
+
+	/**
+	 * @var string
+	 */
+	protected $language_key;
+	/**
+	 * @var Factory
+	 */
+	protected $factory;
+	/**
+	 * @var array
+	 */
+	protected $not_capitalize = [
+		"en" => [
+			// conjunctions
+			"after",
+			"although",
+			"and",
+			"as far as",
+			"as how",
+			"as if",
+			"as long as",
+			"as soon as",
+			"as though",
+			"as well as",
+			"as",
+			"because",
+			"before",
+			"both",
+			"but",
+			"either",
+			"even if",
+			"even though",
+			"for",
+			"how",
+			"however",
+			"if only",
+			"in case",
+			"in order that",
+			"if",
+			"neither",
+			"nor",
+			"once",
+			"only",
+			"or",
+			"now",
+			"provided",
+			"rather",
+			"than",
+			"since",
+			"so that",
+			"so",
+			"than",
+			"that",
+			"though",
+			"till",
+			"unless",
+			"until",
+			"when",
+			"whenever",
+			"where as",
+			"wherever",
+			"whether",
+			"while",
+			"where",
+			"yet",
+			// prepositions
+			"about",
+			"above",
+			"according to",
+			"across",
+			"after",
+			"against",
+			"along with",
+			"along",
+			"among",
+			"apart from",
+			"around",
+			"as for",
+			"as",
+			"at",
+			"because of",
+			"before",
+			"behind",
+			"below",
+			"beneath",
+			"beside",
+			"between",
+			"beyond",
+			"but",
+			"by means of",
+			"by",
+			"concerning",
+			"despite",
+			"down",
+			"during",
+			"except for",
+			"excepting",
+			"except",
+			"for",
+			"from",
+			"in addition to",
+			"in back of",
+			"in case of",
+			"in front of",
+			"in place of",
+			"inside",
+			"in spite of",
+			"instead of",
+			"into",
+			"in",
+			"like",
+			"near",
+			"next",
+			"off",
+			"onto",
+			"on top of",
+			"out out of",
+			"outside",
+			"over",
+			"past",
+			"regarding",
+			"round",
+			"on",
+			"of",
+			"since",
+			"through",
+			"throughout",
+			"till",
+			"toward",
+			"under",
+			"underneath",
+			"unlike",
+			"until",
+			"upon",
+			"up to",
+			"up",
+			"to",
+			"with",
+			"within",
+			"without",
+			// articles
+			"a",
+			"an",
+			"few",
+			"some",
+			"the",
+			"one",
+			"this",
+			"that"
+		]
+	];
+
+
+	/**
+	 * CaseOfLabelIfPossible constructor
+	 *
+	 * @param string  $language_key
+	 * @param Factory $factory
+	 */
+	public function __construct(string $language_key, Factory $factory) {
+		$this->language_key = $language_key;
+		$this->factory = $factory;
+	}
+
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @throws LanguageNotSupportedException
+	 */
+	public function transform($from): string {
```

Comment:
Please remove the `string` return type here, because the `Transformation` interface and the `DeriveApplyToFromTransform` trait do not require a return type.

Reply:
ok